balancedBrackets.java

```diff
@@ -0,0 +1,55 @@
+import java.io.*;
+import java.math.*;
+import java.security.*;
+import java.text.*;
+import java.util.*;
+import java.util.concurrent.*;
+import java.util.regex.*;
+
+public class Solution {

```

Comment:
Can you keep either the class name or the file name same because it would throw an error when you will compile the code.